yt/frontends/ramses/data_structures.py

```diff
@@ -857,9 +867,59 @@ def read_namelist(self):
 
             self.parameters["namelist"] = nml
 
+    @staticmethod
+    def _sanitize_filename(path):
+        # Do not forget to resolve so that it works with symlinks
+        filename = Path(path).resolve()
```

Comment:
I would suggest using the same variable name. `filename` matches the line where it's used so it's the easy choice.